lib/spack/spack/cmd/buildcache.py

```diff
@@ -460,13 +460,19 @@ def push_fn(args):
             tty.info("All specs are already in the buildcache. Use --force to overwrite them.")
         else:
             tty.info(
-                "The following {} specs were skipped as they already exist in the buildcache:\n"
-                "    {}\n"
-                "    Use --force to overwrite them.".format(
-                    len(skipped), ", ".join(elide_list([_format_spec(s) for s in skipped], 5))
-                )
+                f"The following {len(skipped)} specs were skipped as they already exist in the"
```

Comment:
```suggestion
                f"The following {llnl.util.string.plural(len(skipped), "spec")} were skipped as they already exist in the"
```
This will properly display "spec" if it's only one and "specs" for multiple.